src/services/Node/index.js

```diff
@@ -29,30 +29,27 @@ class Node {
 
   async createIdentity(keymanager) {
     const node = await this.api();
-    let identity_number = await node.tx.identityModule
+    let retval = null;
+    await node.tx.identityModule
```

Comment:
@isaacadams if you `let retval = await etc` here, you get back the `() => subscribe.unsubscribe()` or whatever it was we were getting. This _does_ get back a correct value from the chain, but it takes it a minute (because events aren't emitted immediately) so we need to _wait_ for this and assign to the return value variable.